app/Livewire/SettingsOauth.php

```
@@ -38,16 +47,48 @@ public function mount()
+            $oauth->fill([
+                'enabled' => $oauthData['enabled'],
+                'client_id' => $oauthData['client_id'],
+                'client_secret' => $oauthData['client_secret'],
+                'redirect_uri' => $oauthData['redirect_uri'],
+                'tenant' => $oauthData['tenant'],
+                'base_url' => $oauthData['base_url'],
+            ]);
+
             if (! $oauth->couldBeEnabled()) {
                 $oauth->update(['enabled' => false]);
                 throw new \Exception('OAuth settings are not complete for '.$oauth->provider.'.<br/>Please fill in all required fields.');
```

PHPMD (2.15.0)

Missing class import via use statement (line '64', column '27'). (undefined)

(MissingImport)